### PR TITLE
Change Nav & Render Markdown Files

### LINKED ISSUE #147: Change Nav and add schedule and challenge pages
image

The final Nav will have 5 links:
- Home (`/`) Home Page
- Events (`/events`) View events during the event
- Schedule (`/schedule`) The generic schedule of key events (basically the opening ceremony and when devpost opens)
- Challenges (`/challenges`) A list of company challenges
- Apply (`/apply`) A link to Obos 

Tasks:
- [x] Change the nav in gigabowser to be the same as the list above
- [x] Create layout html page that will be used to contain the markdown (with YAML front-matter) post (basically it just contains a nav and a bootstrap container). This will be used along side `renderPost` (in `lib/resolvers.js`) to render the markdown files that show the schedule and challenges pages.
- [x] Create schedule markdown file (for now it should just say coming soon or something)
- [x] Create challenges markdown file (for now it should just say coming soon or something)

### PR DESCRIPTION
**Pull request by samarthdave**
Wowow, first of all we've finished so many tickets on this freaking repo 147 dang.

On another note, if people ask me if I'm a minimialist, I show them this: 
image

All bad jokes aside, I just ripped all the colors so we're far from done. I'll try to replicate Portal page best as possible and fix this look. I also changed the Navbar to ```navbar-light``` and removed the background too.

Closes #147

### CODE REVIEW

File: views/partials/md/content.ejs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="container">
```
Comment (geooot) [inline views/partials/md/content.ejs:1]:
```suggestion
<div class="container pt-5">
```

Add some spacing to the top of the container

Comment (geooot) [inline views/partials/md/content.ejs:1]:
```suggestion
<div class="container pt-5">
```

Add some spacing to the top

File: scss/navbar.scss [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+  .navbar-nav {
+    font-size: 1.25em !important;
+  }
 
-.nav-link {
-  padding: 0.5em 1em !important;
-  transition: color 0.25s ease;
-}
+  .nav-link {
+    padding: 0.5em 1em !important;
```
Comment (geooot) [inline scss/navbar.scss:15]:
```suggestion
    padding: 10px 15px !important;
```

Match the nav link spacing exactly

File: views/partials/md/navbar.ejs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<nav class="navbar navbar-expand-sm navbar-light md-layout-nav">
+  <span class="">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-responsive-dropdown" aria-controls="navbar-responsive-dropdown" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <img src="/static/img/logos/main.png" class="md-nav-logo d-none d-sm-block">
+  </span>
+  <div class="justify-content-center collapse navbar-collapse" id="navbar-responsive-dropdown">
+    <ul class="navbar-nav ml-auto mr-auto pt-2">
```
Comment (geooot) [inline views/partials/md/navbar.ejs:9]:
```suggestion
    <ul class="navbar-nav ml-auto mr-auto">
```

Remove offsetting padding (to make the things centered vertically)

File: scss/navbar.scss [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
   z-index: 100;
   padding-top: 1.25em !important;
   top: 0;
-}
 
-.navbar-nav {
-  font-size: 1.25em !important;
-}
+  .navbar-nav {
+    font-size: 1.25em !important;
```
Comment (geooot) [inline scss/navbar.scss:11]:
```suggestion
    font-size: 1.25em !important;
    padding-right: 40px;
```

This is necessary to make the nav perfectly center

**Comment by samarthdave** [comment]
Also, I just added scripts partial for mobile navbar (since navbar needs ```bootstrap.min.js```)

Okay, I found the same problem in the Obos navbar on iPad/tablet like you said. I'll look into it:

image

**Comment by samarthdave** [comment]
Okay, I just changed it from ```navbar-expand-sm``` to ```navbar-expand-lg```.

Also, I want to merge the regular ```navbar.ejs``` with ```/md/navbar.ejs``` partial but the problem is that one has the TD logo. Thoughts?

**Pull request merged**